UIkitDZ/VC/PaymentVC.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  PaymentVC.swift
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 03.07.2021.
+//
+
+import UIKit
+
+class PaymentVC: UIViewController {
+    
+    var nameOfPizza = String()
```

Comment:
```suggestion
  private  var nameOfPizza = String()
```
и все что ниже private